test/passing/module.ml

```diff
@@ -28,3 +28,26 @@ module C = struct
 
   and B : sig end = B
 end
+
+let _ =
+  ( module M : S
+    with type long_long_type = t
+     and type long_long_long_type = u
+     and type very_long_long_type = t )
+
+let _ =
+  ( module struct
+      type t
+  end : S with type t = t )
```

Comment:
I think `end` should start in the same column as `module`. Also can you check the output when there is enough type constraints to break the line in this case?

Reply:
This was a bug, fixed.